setup.py

```diff
@@ -3,23 +3,30 @@
 from io import open
 import os
 import sys
-
+import platform
+from setuptools import Extension, setup
+from Cython.Build import cythonize
 
 here = os.path.abspath(os.path.dirname(__file__))
 
 # Get the long description from the README file
 with open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
-cmake_args = []
-for option in ("CYTHON_LINETRACE", "CYTHON_ANNOTATE"):
-    value = os.environ.get(option)
-    if value is not None and bool(int(value)):
-        cmake_args.append("-D{}:BOOL=TRUE".format(option))
-
-# On Mac, require OSX >= 10.9 so we can use te new libc++ targeting C++11
-if sys.platform == "darwin":
-    cmake_args.append("-DCMAKE_OSX_DEPLOYMENT_TARGET:STRING=10.9")
+extra_args = []
+linetrace = False
+if int(os.environ.get('CYTHON_LINETRACE', '0')):
+    linetrace = True
+    extra_args.append(('CYTHON_TRACE_NOGIL', '1'))
+
+extension = Extension(
+    'uharfbuzz._harfbuzz',
+    define_macros=[('HB_NO_MT', '1')],
+    include_dirs=['harfbuzz/src'],
+    sources=['src/uharfbuzz/_harfbuzz.pyx', 'harfbuzz/src/harfbuzz.cc'],
+    language='c++',
+    extra_compile_args=[] if platform.system() == 'Windows' else ['-std=c++11'],
+)
 
 setup_params = dict(
```

Comment:
```suggestion
setup(
```

And remove the `setup` call below.

Reply:
done

Reply:
oh sorry that I didn't what actually you wanted, applied.